examples/managed_node_groups/main.tf

```diff
@@ -74,6 +74,16 @@ module "eks" {
 
   node_groups = {
     example = {
+      # In version 1.17 of this module, the way that node group names are
```

Comment:
I dont think we require change examples. Adding those comments can be misleading for new users. Lets dont touch examples at all as I dont think is required